apps/dotlottie-web-node-example/index.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2023 Design Barn Inc.
+ */
+
+/* eslint-disable no-secrets/no-secrets */
+
+import fs from 'node:fs';
+import path from 'node:path';
+
+import { DotLottie } from '@lottiefiles/dotlottie-web';
+import type { FrameEvent } from '@lottiefiles/dotlottie-web/src';
```

Comment:
import type { FrameEvent } from '@lottiefiles/dotlottie-web';

is better. why use the src/ path here?

Reply:
Nice catch!!